app/controllers/spree/admin/orders_controller_decorator.rb

```diff
@@ -100,18 +100,17 @@ def update_distribution_charge
   private
 
   def orders
-    if json_request?
-      @search = OpenFoodNetwork::Permissions.new(spree_current_user).editable_orders.ransack(params[:q])
-      @search.result.reorder('id ASC')
-    else
-      @search = Spree::Order.accessible_by(current_ability, :index).ransack(params[:q])
-
-      # Replaced this search to filter orders to only show those distributed by current user (or all for admin user)
-      @search.result.includes([:user, :shipments, :payments]).
-        distributed_by_user(spree_current_user).
-        page(params[:page]).
-        per(params[:per_page] || Spree::Config[:orders_per_page])
-    end
+    @search = if json_request?
+                OpenFoodNetwork::Permissions.new(spree_current_user).editable_orders.ransack(params[:q])
+              else
+                Spree::Order.accessible_by(current_ability, :index).ransack(params[:q])
+              end
```

Comment:
I think this would look better with an extractmethod. The name of the method is an opportunity to describe what is being done: user_orders?
@search = user_orders.ransack(params[:q])
...

```
def user_orders
  if json_request?
      OpenFoodNetwork::Permissions.new(spree_current_user).editable_orders
  else
    Spree::Order.accessible_by(current_ability, :index)
  end
end
```